appconfig/settings/tailer.q

```diff
@@ -3,8 +3,9 @@
 \d .wdb
 savedir:hsym `$getenv[`KDBWDB]		// location to save wdb data
 hdbdir:hsym`$getenv[`KDBHDB]            // move wdb database to different location
-sortworkertypes:()			// tailer doesn't need to connect to sortworkers
-
+sortworkertypes:()			// WDB doesn't need to connect to sortworkers
+rowthresh:10000				// table savedown row size threshold
+period:0D00:10:00			// how often to check table sizes against threshold
 
 \d .servers
-CONNECTIONS:`segmentedtickerplant`sort`gateway`rdb`hdb
+CONNECTIONS:`segmentedtickerplant`sort`gateway`rdb`hdb`tailer_seg1`tailer_seg2`tr_seg1`tr_seg2,`tailer_seg1
```

Comment:
Have you added tailer_seg1 twice?
```suggestion
CONNECTIONS:`segmentedtickerplant`sort`gateway`rdb`hdb`tailer_seg1`tailer_seg2`tr_seg1`tr_seg2
```

Reply:
also - should a given tailer be connecting to all other tailers & tail readers? this seems wrong IMO e.g. tailer_seg1 shouldn't need to connect to tailer_seg2 etc.

Reply:
Yeah i agreee, there shouldn't be a need to specify specific tailer to tailreader connections in this way as there are lines to do it automatically and the tailers don't need to connect to each other

Reply:
Hi Guys, I have updated this tailer.q file but, I have kept the connections for tailer1 and tailreeader1. These connections are here for testing purposes as I connect to both when testing.